tests/test_tf_ngmix.py

```diff
@@ -1,43 +1,127 @@
 # This module tests our implementation against ngmix
 from numpy.testing import assert_allclose
+
 import autometacal
 import ngmix
 import numpy as np
 import tensorflow as tf
-
+import galsim
 scale = .2
 stamp_size=51
 Ngals = 1000
 
+
+def make_data(rng, noise, shear):
+  """
+  simulate an exponential object with moffat psf
+
+  Parameters
+  ----------
+  rng: np.random.RandomState
+    The random number generator
+  noise: float
+    Noise for the image
+  shear: (g1, g2)
+    The shear in each component
+
+  Returns
+  -------
+  ngmix.Observation
+  """
+
+  psf_noise = 1.0e-6
+
+  scale = 0.263
```

Comment:
inconsistent with what's one line 9

Reply:
Yes, this seems to be an introduced mistake by some github mishandling.  I'll fix it promptly.

Reply:
Fixed.